### ISSUE TITLE
Twilio node not persisting token

### ISSUE THREAD

**Issue by fsamir** [node-red/node-red-nodes#100]
When using 'local credentials' on the Twilio node, I get the following error:
`[red] twilio out : RestClient requires an Account SID and Auth Token set explicitly or via the TWILIO_ACCOUNT_SID and TWILIO_AUTH_TOKEN environment variables
[red] unknown type: twilio out`

Some debugging tells me the token isn't getting persisted and `RED.nodes.getCredentials(n.id);` returns undefined.

Thanks

**Comment by fsamir** [comment]
Also, if I define the credentials (below) directly on node-red settings it does work, but "global/local options" are no longer available and the node is marked with an error.
 ```
 twilio: {
            account: '',
            authtoken: '',
            from: ''
        },
```

**Comment by knolleary** [comment]
@fsamir we had fixed this in git but failed to publish the fix to npm.

I've just published 0.0.6 and verified the fix locally.

**Comment by dceejay** [comment]
Hi @fsamir - can we close this issue now ? thanks
